src/main/java/com/ibm/nfvodriver/service/LifecycleManagementService.java

```diff
@@ -37,44 +37,44 @@ public LifecycleManagementService(NSLifecycleManagementDriver nsLifecycleManagem
-                externalMessagingService.sendDelayedExecutionAsyncResponse(new ExecutionAsyncResponse(requestId, ExecutionStatus.COMPLETE, null, outputs, Collections.emptyMap()), properties.getExecutionResponseDelay());
+        try {if("Create".equalsIgnoreCase(executionRequest.getLifecycleName())) {
+            // Generate CreateNSRequest message
+            final String createNsRequest = messageConversionService.generateMessageFromRequest("CreateNsRequest", executionRequest);
+            // Send message to NFVO
+            final String nsInstanceResponse = nsLifecycleManagementDriver.createNsInstance(executionRequest.getDeploymentLocation(), createNsRequest);
+            // Convert response into properties to be returned to ALM
+            final Map<String, Object> outputs = messageConversionService.extractPropertiesFromMessage("NsInstance", executionRequest, nsInstanceResponse);
+            final String requestId = UUID.randomUUID().toString();
+            // Delay sending the asynchronous response (from a different thread) as this method needs to complete first (to send the response back to Brent)
+            externalMessagingService.sendDelayedExecutionAsyncResponse(new ExecutionAsyncResponse(requestId, ExecutionStatus.COMPLETE, null, outputs, Collections.emptyMap()), properties.getExecutionResponseDelay());
+
+            return new ExecutionAcceptedResponse(requestId);
+
+            } else if ("Install".equalsIgnoreCase(executionRequest.getLifecycleName())) {
+                // Instantiate
+                 final String nsInstanceId = executionRequest.getStringResourceProperty("nsInstanceId");
+                 final String instantiateNsRequest = messageConversionService.generateMessageFromRequest("InstantiateNsRequest", executionRequest);
+                 final String requestId = nsLifecycleManagementDriver.instantiateNs(executionRequest.getDeploymentLocation(), nsInstanceId, instantiateNsRequest);
+                 return new ExecutionAcceptedResponse(requestId);
 
-                // Send response back to ALM
-                return new ExecutionAcceptedResponse(requestId);
             } else if ("Configure".equalsIgnoreCase(executionRequest.getLifecycleName())) {
                 // Instantiate
                 final String nsInstanceId = executionRequest.getStringResourceProperty("nsInstanceId");
                 final String instantiateNsRequest = messageConversionService.generateMessageFromRequest("InstantiateNsRequest", executionRequest);
                 final String requestId = nsLifecycleManagementDriver.instantiateNs(executionRequest.getDeploymentLocation(), nsInstanceId, instantiateNsRequest);
                 return new ExecutionAcceptedResponse(requestId);
 
-            } else if ("Start".equalsIgnoreCase(executionRequest.getLifecycleName())) {
-                // Operate (Start)
+            } else if ("Update".equalsIgnoreCase(executionRequest.getLifecycleName())) {
+                // Operate (Update)
```

Comment:
Remove Operate and just keep Update in the commented line

Reply:
changes done and committed the code